tools/universe_builder.py

```diff
@@ -1,102 +1,71 @@
 #!/usr/bin/env python3
 
+import base64
 import difflib
 import hashlib
+import json
 import logging
 import os
 import os.path
 import re
-import shutil
 import sys
 import tempfile
-import time
-import zipfile
+from collections import OrderedDict
```

Comment:
opinion: My preference is avoiding `from` unless it's truly onerous.  Iteratively it costs very little, but when you find yourself debugging a scenario where there are multiple copies of a module in memory, you become very sad.